src/main/java/net/helix/pendulum/controllers/RoundViewModel.java

```diff
@@ -258,45 +253,34 @@ public static int getRoundIndex(TransactionViewModel milestoneTransaction) {
                 Set<Hash> prevMilestones = prevMilestone.getHashes();
                 List<List<Hash>> merkleTree = Merkle.buildMerkleTree(new ArrayList<>(prevMilestones));
                 if (transaction.getTrunkTransactionHash().equals(merkleTree.get(merkleTree.size() - 1).get(0))) {
-                    //System.out.println("trunk (prev. milestones): ");
```

Comment:
Let's convert this into `log.trace()`

Reply:
there is no logger instance in controllers!